lib/lob/resources/resource_base.rb

```diff
@@ -71,16 +70,20 @@ def submit(request)
         if query != {}
           url = "#{url}?#{build_nested_query(query)}"
         end
-        
+
+        if body&[:merge_variables]&.class == Hash
+          body[:merge_variables] = body[:merge_variables].to_json()
+        end
+
+        client_params = {
+          headers: headers,
+          method: request[:method],
+          url: url,
+        }
+        client_params[:body] = body if body
+
         begin
-          if method == :get || method == :delete
```

Comment:
Can we wrap the assignment to `client_params[:body]` with these 2 cases please?

GET and DELETE operations are intended by specification to only leverage the URI. Removing this case enables passing a body in these requests. Although it isn't specifically prohibited it the spec, it is intended to be functionally irrelevant.

Ref: [https://www.rfc-editor.org/rfc/rfc2616#section-9.3](https://www.rfc-editor.org/rfc/rfc2616#section-9.3)